spec/game_statistics_spec.rb

```diff
@@ -45,7 +45,19 @@
 
   describe "#percentage_home_wins" do
     it "returns a percentage of games that a home team has won as a float (rounded to the nearest 100th)" do
-      # expect(@game_stats.percentage_home_wins).to be_a(Float)
+      expect(@game_stats.percentage_home_wins).to be_a(Float)
+    end
```

Comment:
we should also make the assertion:
`expect(@game_stats.percentage_home_wins).to eq(0.44)`

Reply:
Updated